kong/core/handler.lua

```diff
@@ -86,7 +86,7 @@ return {
 
       if api.https_only and not utils.check_https(api.http_if_terminated) then
         ngx.header["connection"] = "Upgrade"
-        ngx.header["upgrade"]    = "TLS/1.0, HTTP/1.1"
+        ngx.header["upgrade"]    = "TLS/1.2, HTTP/1.1"
```

Comment:
As far as I understand, the `Upgrade` header recommends a protocol that the client "may" use, and we recommend `TLS/1.2`. He can still use `TLS/1.1` since it's supported by the server, but we would still recommend the latest one.